<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>domain7</class>
 <widget class="QWidget" name="domain7">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>316</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>preamble: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lEditpreamble"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>id: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lEditid"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>type: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lEdittype"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>rate1: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lEditrate1"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>rate2: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lEditrate2"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="btnConfirm7">
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
